src/simulation/m_start_up.fpp

```diff
@@ -1259,17 +1270,29 @@ contains
         print *, 'Computed derived vars'
 #endif
 
+        time_prev = mytime
         mytime = mytime + dt
 
-        ! Total-variation-diminishing (TVD) Runge-Kutta (RK) time-steppers
-        if (time_stepper == 1) then
-            call s_1st_order_tvd_rk(t_step, time_avg)
-        elseif (time_stepper == 2) then
-            call s_2nd_order_tvd_rk(t_step, time_avg)
-        elseif (time_stepper == 3 .and. (.not. adap_dt)) then
-            call s_3rd_order_tvd_rk(t_step, time_avg)
-        elseif (time_stepper == 3 .and. adap_dt) then
-            call s_strang_splitting(t_step, time_avg)
+        if(lag_bubbles) then 
```

Comment:
is this option really 'forced' to a user using E-L bubbles? no clean and easy way to leave the user with the fall back option of a regular time stepper? if we can give them all options, that would be very handy (and we can also abstract out the lag_bubble part of the time stepper routine here into its own subroutine).

Reply:
I have defined time_stepper == 4 to activate the RKCK time stepper, and added the Lagrange subroutines into the regular TVD RK steppers. With this, I have testes these changes using the bubble screen problem using different time-steppers.
![Comparison-PressureEvolution](https://github.com/user-attachments/assets/c605f0fb-bc7d-4e30-b004-a48309bd9cec)

Reply:
so it looks like the method works fine with 2nd/3rd order RK? if so, it appears that the code still restricts EL bubbles to RK45?

Reply:
fixed :) I defined a time_stepper = 4 to activate the rkck algorithm.